README.md

```diff
@@ -51,7 +51,7 @@ The rover can be operated through Mission Control with either a keyboard or two
 ![Armo controls](/src/components/help/armControls.png)
 ![Keyboard controls](/src/components/help/keyboardControls.png)
 
-## Messages (`v2024.1.1`)
+## Messages (`v2024.1.2`)
```

Comment:
You didn't change the protocol, right? In that case you don't need to bump the version number. This is just saying what version the protocol is.